patches/747-mikrotik-extra-support.patch

```diff
@@ -277,6 +286,7 @@
  	zyxel,wre6606)
  		ucidef_set_interface_lan "eth0"
  		;;
++ mikrotik,discg-5acd*|\
```

Comment:
Indentation looks wrong here.

Reply:
I'm not certain what to do here, I see the difference in this view, but when I go to the edit view, the indentation looks consistent...
![Indentation in editor](https://github.com/aredn/aredn/assets/44928842/41892163-4a71-4025-8912-abd8a597c1e8)

Reply:
I assume it's a spaces vs tabs thing. Just use the same as they do.

Reply:
I just went back in the editor, deleted the space, and inserted a tab. I hope that fixes it...